src/archive.cpp

```diff
@@ -1,5 +1,6 @@
 #include "metadata-parser.h"
 #include "archive.h"
+#include <fstream>
```

Comment:
Same comment

Reply:
Same as the previous comment. here it's needed by line 62.  I'll add a documentation in the PR message.